css/templates/gallery/page.css

```diff
@@ -21,9 +21,85 @@
 @import '../../core.css';
 
 @import './page-vars';
+@import './components/_landing-images';
 
 body {
   color: var(--font-color-primary);
   font-family: var(--font-family-primary);
   line-height: var(--line-height);
 }
+
+.gallery-base-content {
+  background: #fff;
+  overflow-x: hidden;
+}
+
+.gallery-base-container {
+  position: relative;
+  width: calc((23 / 50) * 100vw + (150 / 16) * 1rem);
+  max-width: calc((800 / 16) * 1rem);
+}
+
+.gallery-base-invert {
+  color: #fff;
+  mix-blend-mode: difference;
+  -webkit-font-smoothing: antialiased;
+}
+
+/* Hack media query to target IE, which does not support mix-blend-mode. */
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
move this to the bottom, also i am not sure if :none is a value that is valid

https://msdn.microsoft.com/en-us/library/hh771830(v=vs.85).aspx

Reply:
Neal might be able to help here @10xjs